src/parseSet.java

```diff
@@ -0,0 +1,71 @@
+package src;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.*;
+
+class parseSet {
+
+    private static final String DELIMITER = "[^a-zA-Z0-9_,]+";
+    private static final String FIRSTPATH = "src/FIRST.txt";
+    private static final String PREDICTPATH = "src/PREDICT.txt";
+    static Map<String, Map<String, Boolean>> FIRST = new HashMap<>();
+    static Map<String, Map<String, List<String>>> PREDICT = new HashMap<>();
+
+    /**
+     * Populate FIRST or PREDICT data structure for LL parsing given file with
```

Comment:
(For the whole file) Why is this in its own file, as opposed to in the parse file?